### PR TITLE
EZP-31236: Added title to action buttons

### PR DESCRIPTION
**Pull request by lucasOsti**
Question      | Answer
| ------------- | ---
| Tickets       | https://jira.ez.no/browse/EZP-31236
| Bug fix?      | no
| New feature?  | yes
| BC breaks?    | no
| Tests pass?   | yes
| Doc needed?   | no
| License       | [GPL-2.0](https://github.com/ezsystems/ezplatform-admin-ui/blob/master/LICENSE)

Add title to action buttons
All PR’s to this ticket:
https://github.com/ezsystems/ezplatform-matrix-fieldtype/pull/30
https://github.com/ezsystems/ezplatform-form-builder/pull/205
https://github.com/ezsystems/ezplatform-user/pull/58
https://github.com/ezsystems/ezplatform-workflow/pull/106
https://github.com/ezsystems/date-based-publisher/pull/148
https://github.com/ezsystems/ezplatform-admin-ui-modules/pull/246
https://github.com/ezsystems/ezplatform-admin-ui/pull/1176

#### Checklist:
- [x] Coding standards (`$ composer fix-cs`)
- [x] Ready for Code Review

### CODE REVIEW

File: src/lib/EventListener/UserPasswordChangeRightSidebarListener.php [hunk 1]
```diff
@@ -40,15 +49,31 @@ public function onUserPasswordChangeRightSidebarConfigure(ConfigureMenuEvent $ev
                 'attributes' => [
                     'class' => 'btn--trigger',
                     'data-click' => '#user_password_change_change',
+                    'data-extra-classes' => 'ez-tooltip--medium',
+                    'data-placement' => 'left',
+                    'title' => $this->translator->trans(
+/** @Ignore */ self::ITEM__UPDATE,
```
Comment (GrabowskiM) [inline src/lib/EventListener/UserPasswordChangeRightSidebarListener.php]: No tabs here?

File: src/lib/EventListener/UserPasswordChangeRightSidebarListener.php [hunk 2]
```diff
@@ -40,15 +49,31 @@ public function onUserPasswordChangeRightSidebarConfigure(ConfigureMenuEvent $ev
...
         );
         $menu->addChild(
             self::ITEM__CANCEL,
             [
-                'route' => 'ezplatform.dashboard',
+                'attributes' => [
+                    'data-extra-classes' => 'ez-tooltip--medium',
+                    'data-placement' => 'left',
+                    'title' => $this->translator->trans(
+/** @Ignore */ self::ITEM__CANCEL,
```
Comment (GrabowskiM) [inline src/lib/EventListener/UserPasswordChangeRightSidebarListener.php]: and here?

File: src/lib/EventListener/UserPasswordChangeRightSidebarListener.php (hunk 1 above)
Comment (lucasOsti) [reply src/lib/EventListener/UserPasswordChangeRightSidebarListener.php]: This is effect of "composer fix-cs"

File: src/lib/EventListener/UserPasswordChangeRightSidebarListener.php (hunk 2 above)
Comment (lucasOsti) [reply src/lib/EventListener/UserPasswordChangeRightSidebarListener.php]: This is effect of composer fix-cs

**Pull request merged**